artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -976,10 +1052,29 @@ components:
         application/json:
           schema:
             $ref: "#/components/schemas/ErrorInfo"
-          example:
-            status: 500
-            code: "INTERNAL"
-            message: "Server error"
+          examples:
+            GENERIC_500_INTERNAL:
+              description: Problem in Server side. Regular Server Exception
+              value:
+                status: 500
+                code: INTERNAL
+                message: Unknown server error. Typically a server bug.
+    Generic501:
```

Comment:
I would remove as per my previous comment. A subscription service MUST be complete, if not there is no such subscription service

Reply:
Yes ! Will do